presto-product-tests/conf/docker/singlenode-kerberos-impersonation/docker-compose.yml

```diff
@@ -15,20 +15,21 @@ services:
   presto-master:
     hostname: presto-master
     # cdh5-hive-kerberized contains keytabs and the `krb5.conf` within
-    image: 'teradatalabs/cdh5-hive-kerberized:2.6.0-cdh5.6.0-sshd-4e54a19'
+    image: 'teradatalabs/cdh5-hive-kerberized:2'
     depends_on:
       - 'hadoop-master'
     command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-kerberized run
     volumes:
       - ../../../../:/docker/volumes/presto
-      - ../../../../presto-product-tests/conf/presto/etc/hive-kerberized.properties:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
+      - ../../../../presto-product-tests/conf/presto/etc/hive-kerberized-with-impersonation.properties:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
```

Comment:
I think we should keep names consistent. If we call environment `singlenode-kerberos-no-impersonation` than we should probably call the `hive.properties` file in a similar way, say `hive-kerberos-no-impersonation.properties`.